source/_integrations/statistics.markdown

```diff
@@ -44,10 +44,12 @@ The following characteristics are supported for `sensor` source sensors:
 | `average_timeless` | The average value of stored measurements. This method assumes that all measurements are equally spaced and, therefore, time is ignored and a simple average of values is computed. Equal to `mean`.
 | `change_sample` | The average change per sample. The difference between the oldest and newest measurement is divided by the number of in-between measurements (n-1).
 | `change_second` | The average change per second. The difference between the oldest and newest measurement is divided by seconds between them.
-| `change` | The difference between the oldest and newest measurement stored.
+| `change` | The difference between the oldest and newest measurement.
 | `count` | The number of stored source sensor readings. This number is limited by `sampling_size` and can be low within the bounds of `max_age`.
-| `datetime_newest` | The timestamp of the newest measurement stored.
-| `datetime_oldest` | The timestamp of the oldest measurement stored.
+| `datetime_newest` | The timestamp of the newest measurement.
```

Comment:
In this case, should the source sensor have device class timestamp? Maybe add a column to the table with sensor type, or (and that's probably clearer) add a new section "Timestamp source sensor"?